Documentation/Migration/Index.rst

```diff
@@ -14,21 +14,21 @@ Migration: From Sphinx to PHP-based rendering
     Text rendering to try it out. The new rendering will become mandatory in
     August 2024.
 
-The main difference that concerns you is that the PHP-based rendering
-requires a file called :file:`Documentation/guides.xml` for configuration
-while in Sphinx rendering a file called :file:`Documentation/Settings.cfg` was
-used. In the transition period we detect if a file called
-:file:`Documentation/guides.xml` is present and then switch to the
+The main difference compared to the Sphinx rendering is that the PHP-based rendering
+requires a file called :file:`Documentation/guides.xml` for configuration.
+The Sphinx rendering required a file called :file:`Documentation/Settings.cfg`.
+In the transition period the process detects if a file called
```

Comment:
Actually, I don't like the word "process" here. It sounds like you can run the guides rendering locally also for Settings.cfg, which is not the case. It is only related to the official rendering with our GitHub action.

Reply:
How about my suggestions or what would you suggest?